### DISCUSSION TITLE
🐞 /message/xxxx/delete This interface is invalid

### DISCUSSION THREAD

**Discussion by fixalllow** [aldinokemal/go-whatsapp-web-multidevice#183] [Announcements]
### What happened?

I tried deleting the message from my device after sending it, but it didn't work

`http://127.0.0.1:3000/message/3EB06A9D41EBEDE0273CBC/delete1` POST

data 

```
{"phone":"xxxxxxx@s.whatsapp.net"}
```

res

```
{"code":"SUCCESS","message":"Message deleted successfully"}
```

### Reproduction Steps

```bash
I tried deleting the message from my device after sending it, but it didn't work
```

### Application Version

4.16.0

### Verification

- [X] I have checked for existing closed issues and discussions.

**Comment by fixalllow** [comment]
Help me🥹🥹🥹🥹 

@aldinokemal

**Comment by fixalllow** [comment]
Is the author still maintaining this repository? Why is there no response or solution to bugs?

**Comment by aldinokemal** [comment]
you can try using another account first, 
I just tested and still working no issue
I will move to discussion
thanks

**Reply by fixalllow** [reply]
What I mean is that I sent a message to a friend, and my phone can see that the message was sent. Then I called the delete message interface, and the interface returned a successful deletion, but the message can still be seen on my phone!
